tasks/multipathd.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+- name: Check /etc/multipath.conf
+  stat:
+    path: /etc/multipath.conf
+  register: stat_multipath_conf
+
+- name: Check service multipathd
+  service:
+    name: multipathd
+  register: service_multipathd
+
+- name: Add blacklist to multipath.conf
+  blockinfile:
+    block: |
+      blacklist {
+              devnode "^(ram|raw|loop|fd|md|dm-|sr|scd|st)[0-9]*"
+              devnode "^hd[a-z]*"
+              devnode "^cciss!c[0-9]d[0-9]*[p[0-9]*]"
```

Comment:
Is this cciss blacklist really needed?
Sounds HP specific to me

Reply:
I have to admit I was not sure what those are needed for; I just used the recommendations from the red hat ticket :)

Reply:
My guess is they try to blacklist all devices not needed for the ovirt/rhv node ie all local storage.

Reply:
That is OK. If it works it works.